v1/AlertmanagerConfig.dhall

```diff
@@ -0,0 +1,32 @@
+let imports = ../imports.dhall
+
+let Kubernetes = imports.Kubernetes
+
+let AlertmanagerConfigSpec = ./AlertmanagerConfigSpec.dhall
+
+let AlertmanagerConfig
+    : ∀(_params : { routes : Type }) →
+        { Type : Type
+        , default : { metadata : Optional Kubernetes.ObjectMeta.Type }
+        }
+    = λ(_params : { routes : Type }) →
+        let AlertmanagerConfigSpec =
+              AlertmanagerConfigSpec { routes = _params.routes }
+
+        in  { Type =
+                { metadata : Optional Kubernetes.ObjectMeta.Type
+                , spec : AlertmanagerConfigSpec.Type
+                }
+            , default.metadata = None Kubernetes.ObjectMeta.Type
+            }
```

Comment:
`AlertmanagerConfig` should get `AlertmanagerConfigSpec.Type` as a parameter, not the `routes`